SmartDeviceLinkTests/ProxySpecs/SDLSecondaryTransportManagerSpec.m

```diff
@@ -491,44 +496,67 @@ + (void)swapConnectionMethods {
                 testPrimaryProtocol = [[SDLProtocol alloc] init];
                 testPrimaryTransport = [[SDLTCPTransport alloc] init];
                 testPrimaryProtocol.transport = testPrimaryTransport;
+
                 dispatch_sync(testStateMachineQueue, ^{
                     [manager startWithPrimaryProtocol:testPrimaryProtocol];
                 });
 
                 manager.secondaryTransportType = SDLTransportSelectionIAP;
             });
 
-            it(@"should transition to Connecting state", ^{
+            it(@"should transition to Connecting state but before RAIR is sent", ^{
                 // setToState cannot be used here, as the method will set the state after calling didEnterStateConfigured
                 dispatch_sync(testStateMachineQueue, ^{
                     [manager.stateMachine transitionToState:SDLSecondaryTransportStateConfigured];
                 });
 
                 expect(manager.stateMachine.currentState).to(equal(SDLSecondaryTransportStateConnecting));
+                expect(manager.secondaryProtocol.securityManager).to(beNil());
+
                 OCMVerifyAll(testStreamingProtocolDelegate);
             });
+
+            it(@"should transition to Connecting state after RAIR is sent", ^{
```

Comment:
I'm not sure what this is testing? That it transitions to the Connecting state if the security manager is set and the library becomes ready? I don't know that that's worth testing.